Samples/WebDemo/Controllers/EvaluationController.cs

```diff
@@ -1,49 +1,57 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System;
+using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using Microsoft.PowerFx;
 using Microsoft.PowerFx.Types;
 using PowerFxService.Model;
-using System;
 
 namespace PowerFxService.Controllers
 {
-    // Called by the demo page to evaluate an expression. 
+    // Called by the demo page to evaluate an expression.
     [ApiController]
     [Route("eval")]
     public class EvaluationController : ControllerBase
     {
         private readonly ILogger<EvaluationController> _logger;
-        
+        public static TimeSpan timeout = TimeSpan.FromSeconds(2);
```

Comment:
>timeout [](http://example.com/codeflow?start=31&length=7)

coding conventions comment - fields should start with a '_'.

Reply:
done!